src/configs/config.json

```diff
@@ -8,7 +8,7 @@
             "LIQUIDITY_RANGE_OFFSET": 0.5,
             "LIQUIDITY_ADJUST_THRESHOLD": 0.1
         },
-        "vPERP": {
```

Comment:
there is no `vPERP` on staging, use `vETH` to support both production and staging